.zuul.d/jobs.yaml

```diff
@@ -3,14 +3,20 @@
     name: ansible-runner-build-container-image
     parent: ansible-build-container-image
     description: Build ansible-runner container image
-    pre-run: playbooks/ansible-runner-build-container-image/pre.yaml
+    required-projects:
+      - github.com/ansible/ansible
     timeout: 2700
     provides: ansible-runner-container-image
+    requires:
+      - python-builder-container-image
+      - python-base-container-image
```

Comment:
what's the need for this image, since it seems like it's a subset of `python-builder-container-image` after a quick look at the Containerfiles

Reply:
python-builder is for compile time, and python-base is the runtime stage.  The OS for each is the same, just different dependencies, and build scripts installed.